src/restic/node.go

```diff
@@ -581,6 +581,7 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	case "file":
 		node.Size = uint64(stat.size())
 		node.Links = uint64(stat.nlink())
+		node.Device = uint64(stat.dev())
```

Comment:
This would mean we need two fields: one for st_dev and one st_rdev.
My suggestion would be to have Device for st_dev and Rdevice for st_rdev.